docs/add-ins/addin-tutorial.md

```diff
@@ -304,7 +304,15 @@ Since you previously installed the add-in from a file, you must reinstall it in
 
 1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to reinstall the add-in using the updated **manifest.xml** file.
 
-After you've completed these steps, you should see two new buttons in the ribbon of the message compose window: **Insert gist** and **Insert default gist**. 
+After you've reinstalled the add-in, you can verify that it installed successfully by checking for the options **Insert gist** and **Insert default gist** in the message compose window. Note that nothing will happen if you select either of these items, because you haven't yet specified the code that runs when these items are selected.
```

Comment:
Nit: is "options" the right word? Should it be buttons, since there's an "options" menu tab?

Reply:
Yeah, I struggle with what to call this, because:
- in Windows, **Insert gist** and **Insert default gist** are buttons in the ribbon
- in Outlook on the web, **Insert gist** and **Insert default gist** are items in the pop-up menu when you click the (single/unlabeled) add-in button

Any suggestions for alternate phrasing?

Reply:
maybe call them "commands"

Reply:
For clarity, I've removed the words "the options" from this sentence.

Reply:
Just seeing your comment @ElizabethSamuel-MSFT -- I like that suggestion. I'll update accordingly.

Reply:
@kbrandl That's a good call. More concise and just as informative ("do you see these things?")